modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb

```diff
@@ -54,6 +55,18 @@ def evss_mapper_service(auto_claim)
           ClaimsApi::V2::DisabilityCompensationEvssMapper.new(auto_claim)
         end
 
+        def track_pact_counter(claim)
+          return unless form_attributes['disabilities']&.map { |d| d['isRelatedToToxicExposure'] }&.include? true
+
+          # Fetch the claim by md5 if it doesn't have an ID (given duplicate md5)
+          if claim.id.nil? && claim.errors.find { |e| e.attribute == :md5 }&.type == :taken
+            claim = ClaimsApi::AutoEstablishedClaim.find_by(md5: claim.md5) || claim
+          end
+
+          ClaimsApi::ClaimSubmission.create claim_id: claim.id, claim_type: 'PACT',
+                                            consumer_label: token.payload['label'] || token.payload['cid']
+        end
```

Comment:
So, I think we need the VBMS id based on the ticket: "This DB entry will have to be created in the job as we'll need the ID that VBMS provides.", and we will need to more this to a worker.

Reply:
I think there was some discussion had around this and we didn't update the ticket to reflect, this could be causing some of the miscommunication. I _think_ what was decided is that we could pull the evss_id from the Claim that the submission belongs to and it would achieve the same thing. Right @acovrig ? Am I recollecting correctly? The ID we get back when we submit the claim to the EVSS Container is ultimately the `vbms_id` discussed in the original ticket.

Also, rails is smart enough to just pass the claim to `claim:` and you'll get that dirty work handed to ya.

something like 
```cool_submission_name = ClaimsApi::ClaimSubmission.create claim: claim, claim_type: 'PACT', ........```

then you would have access to the "vbms_id" by

```cool_submission_name.claim.evss_id```